components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/datasource/QueryExecutor.java

```diff
@@ -67,8 +74,8 @@ public boolean executeInsertQuery(String uuid, byte[] businessRule, int deployme
             result = statement.execute();
             return result;
         } catch (SQLException e) {
-            log.error("Inserting business rule " + new String(businessRule, Charset.forName("UTF-8")) +
-                    " is failed due to " + e.getMessage
+            log.error("Inserting business rule " + new String(businessRule, Charset.forName("UTF-8")) + " is failed " +
+                    "due to " + e.getMessage
```

Comment:
Please format properly.

Reply:
This is fixed in master

Reply:
This is fixed in master

Reply:
Please add the commit id, so people can review the changes you have done.

Reply:
This was fixed by 2c0119c4c48e1077512e685493252c388e68a520